cmd/jwt_test.go

```diff
@@ -35,16 +35,10 @@ func testAuthenticate(authType string, t *testing.T) {
 	}{
 		// Access key too small.
 		{"user", "pass", errInvalidAccessKeyLength},
-		// Access key too long.
-		{"user12345678901234567", "pass", errInvalidAccessKeyLength},
 		// Access key contains unsuppported characters.
 		{"!@#$", "pass", errInvalidAccessKeyLength},
-		// Success when access key contains leading/trailing spaces.
-		{" " + serverCred.AccessKey + " ", serverCred.SecretKey, errInvalidAccessKeyLength},
 		// Secret key too small.
 		{"myuser", "pass", errInvalidSecretKeyLength},
-		// Secret key too long.
-		{"myuser", "pass1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890", errInvalidSecretKeyLength},
```

Comment:
There is actually no difference between the last Success test and having more tests with different length access or secret keys. 
I mean, as long as the keys are longer than minimum value, they should be fine. 
So, what I changed is to have longer than before access and secret keys recorded as valid values so that the last Success case can use these values instead of the default 20/40 lengths.
Another thing I've noticed is that the "unsupported characters" test case actually still tests the length. I also checked the error messages to see if there is an "invalid characters" error message to see if we capture it and there was none. So, I also removed it since it doesn't make sense.